project/Rat.scala

```diff
@@ -39,34 +39,34 @@ object Rat {
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/input18.exisa"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/input19.txt"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/inputBig1M.txt"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/prefix.txt"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/test_DFDL-714.txt"),
-    file("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/uuid.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/hextest.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input1.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input2.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input3.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input4.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input5.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input6.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input7.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input8.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input9.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input10.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input11.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input12.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input13.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input14.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input14.exi"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input14.exisa"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input15.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input16.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input18.json"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input18.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input18.exi"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input18.exisa"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/input19.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/inputBig1M.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/prefix.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/test_DFDL-714.txt"),
+    file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input/uuid.txt"),
```

Comment:
I know you aborted this pull request, but I want to note that you can ignore every file in the `daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input` directory with a single line, `file("daffodil-cli/src/test/resources/org/apache/daffodil/CLI/input")`.  Most of the files are .txt files and you can move the 3 .py/.xml files (the only files which have licenses) to another directory if you feel like keeping them apart.

Reply:
I wonder if we could/should adopt a file naming convention for files that are ALv2 but cannot have a license header in them? Then we can just ignore all files that match that pattern and remove most of this list, and we don't have to worry about accidentally ignoring entire directories that might have a mix of licenses.